base/applications/sndvol32/dialog.c

```diff
@@ -196,7 +191,7 @@ AddDialogControl(
     }
     else
     {
-        /* add data offset */
+        /* skip past additional data */
         Offset += *Offset;
     }
```

Comment:
> what would that look like?

I meant double-check as a human task to do, before documenting/fixing code as need be.
If "you" don't do it in this PR/Issue, please add a `/* FIXME: Check/Document/Fix whether this should be "Offset += 1 + *Offset" to explicitly skip the data count too. */` and maybe file an issue too.

> the code below that does alignment may fix this automatically

It may, except when `Offset` is already aligned.

> if it works, don't change it.

Actually, these alignment lines are inconsistent: comment says "word", code does "double-word".
If we assume current code is correct (as in data count is actually in words, not bytes), then the comment should be fixed.
If we do not, then this needs a FIXME too...